internal/impl/envpath/pathlists.go

```diff
@@ -35,3 +36,22 @@ func JoinPathLists(pathLists ...string) string {
 	}
 	return strings.Join(cleaned, string(filepath.ListSeparator))
 }
+
+func RemoveFromPath(path, pathToRemove string) string {
+	paths := strings.Split(path, string(os.PathListSeparator))
+
+	// Create a new slice to store the modified paths
+	var newPaths []string
+
+	// Iterate through the paths and add them to the newPaths slice if they are not equal to pathToRemove
+	for _, p := range paths {
+		if p != pathToRemove {
+			newPaths = append(newPaths, p)
+		}
+	}
+
+	// Join the modified paths using ":" as the delimiter
+	newPath := strings.Join(newPaths, string(os.PathListSeparator))
```

Comment:
nit: can use `filepath.Join`

Reply:
line 37 too :)

Reply:
`filepath.Join` is for directory paths! not for path lists!

e.g. /path/to/my/file vs path1:path2:path3

Reply:
lol